README.md

```diff
@@ -60,7 +65,7 @@ For prerequisites, installation and setup, see the [Installation and setup](docs
 Currently the Unity GDK is in its early development, so there is no migration path from the [Unity SDK](https://github.com/spatialos/UnitySDK) to the Unity GDK. There will be a migration path in later releases.
 
 ## Roadmap
-See the [latest roadmap](docs/latest-roadmap.md) and let us know what you'd like to see in there.
+See the [latest roadmap Trello board](https://trello.com/b/29tMKyQC) and let us know what you'd like to see in there.
```

Comment:
Before this goes in can we sit down as a team and fill out this Trello board?

Reply:
Do you want to check with @ernopp on this. Will hold the PR - then.

Reply:
Discussed with @ernopp. All good, I think you may need to merge with master before running CI again :)